tests/pyccel/test_pyccel.py

```diff
@@ -652,6 +652,29 @@ def test_arrays_view(language):
             [int] * 6 + [int] * 10
     pyccel_test("scripts/arrays_view.py", language=language, output_dtype=types)
 
+def test_return_numpy_arrays(language):
+    types = [int]*4 # 4 ints for a
+    types += [int]*2 # 2 ints for b
+    types += [float]*2 # 2 floats for c
+    types += [bool]*2 # 2 bools for d
+    types += [complex]*2 # 2 complexs for e
+    types += [int]*5 # 5 ints for f
```

Comment:
`f` contains floats

Reply:
@MostafaMamouni `h` is created in `multi_returns` with the command `x = np.ones(5)`. The default dtype is float so `h` contains floats not ints